src/lib/src/component/navigation/navigation.component.test.setup.ts

```diff
@@ -60,21 +69,18 @@ export function setTestExecutionServiceResponse(service: TestExecutionService, s
   when(service.execute(tclFile.path)).thenReturn(Promise.resolve(response));
 }
 
-export function setTestStatusServiceResponse(service: TestExecutionService, statusCode: number) {
-  const responseBeforeTermination = new Response(new ResponseOptions({
-    status: statusCode,
-    body: 'RUNNING'
-  }));
-  const responseAfterTermination = new Response(new ResponseOptions({
-    status: statusCode,
-    body: 'SUCCESS'
-  }));
+export function setTestStatusServiceResponse(service: TestExecutionService) {
```

Comment:
'set' does not seem to be the best prefix here, what do you think? Maybe something like 'mockTestStatusServiceWithRunningRunningSuccessSequence'? It is quite long, but might help understand its working.